apps/ledger-live-mobile/src/newArch/features/WalletSync/hooks/useRemoveMember.ts

```diff
@@ -59,20 +59,18 @@ export function useRemoveMember({ device, member }: Props) {
         throw new Error("trustchain or memberCredentials is not set");
       }
       try {
-        await runWithDevice(device.deviceId, async transport => {
-          const newTrustchain = await sdk.removeMember(
-            transport,
-            trustchain,
-            memberCredentials,
-            member,
-            {
-              onStartRequestUserInteraction: () => setUserDeviceInteraction(true),
-              onEndRequestUserInteraction: () => setUserDeviceInteraction(false),
-            },
-          );
+        const newTrustchain = await sdk.removeMember(
+          device.deviceId,
+          trustchain,
+          memberCredentials,
+          member,
```

Comment:
( 👍  on mobile we had cleaner code I hope we can also reach on LLD, see here, no casting )